src/topojson/topojson.ml

```diff
@@ -38,12 +38,31 @@ module Make (J : Intf.Json) = struct
         "geometries";
       ]
 
+    let keys_in_use_for_point =
+      [
+        "type";
+        "properties";
+        "coordinates";
+        "bbox";
+        "id";
+        "objects";
+        "geometries";
+      ]
+
     let foreign_members_of_json json =
       match J.to_obj json with
       | Ok assoc ->
           List.filter (fun (k, _v) -> not (List.mem k keys_in_use)) assoc
       | Error _ -> []
 
+    let foreign_members_of_json_for_point json =
```

Comment:
Why not keep one implementation of `foreign_members_of_json` and pass in an extra argument called `keys_in_use` instead?

Reply:
That is true, please take a look at the changes i made. Is it correct?